core/src/main/java/io/confluent/kafka/schemaregistry/rest/extensions/SchemaRegistryResourceExtension.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.kafka.schemaregistry.rest.extensions;
+
+import javax.ws.rs.core.Configurable;
+
+import io.confluent.kafka.schemaregistry.rest.SchemaRegistryConfig;
+import io.confluent.kafka.schemaregistry.storage.KafkaSchemaRegistry;
+
+public interface SchemaRegistryResourceExtension {
+
+  void register(Configurable<?> config,
+                SchemaRegistryConfig schemaRegistryConfig,
+                KafkaSchemaRegistry schemaRegistry);
+
+  void clean();
```

Comment:
Yeah, too late. It's fine, it's a minor API wart, we have more than one of those :)